src/jit/lsraarm64.cpp

```diff
@@ -994,10 +995,44 @@ void LinearScan::TreeNodeInfoInitSIMD(GenTreeSIMD* simdTree, TreeNodeInfo* info)
 void LinearScan::TreeNodeInfoInitHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree, TreeNodeInfo* info)
 {
     NamedIntrinsic intrinsicID = intrinsicTree->gtHWIntrinsicId;
-    info->srcCount += GetOperandInfo(intrinsicTree->gtOp.gtOp1);
-    if (intrinsicTree->gtGetOp2IfPresent() != nullptr)
+
+    GenTreeArgList* argList = nullptr;
+    GenTree*        op1     = intrinsicTree->gtOp.gtOp1;
+    GenTree*        op2     = intrinsicTree->gtOp.gtOp2;
+
+    if (op1->OperIs(GT_LIST))
```

Comment:
`OperIsList()`?

Reply:
Why?

Reply:
It is an explicit helper method for checking `OperIs(GT_LIST)`. We have a few of them and they look to be the most prevalent in the codebase (or at least the most prevalent in the code I've touched so far).

Reply:
Why?  OperIs() is pretty standard in Arm64 lower.

Reply:
FWIW I don't think we have a guidelines or even a lot of consistency on this. `OperIs()` is very useful for making checks for multiple oper values more concise and easier to read. For a single value, I think either is fine.

Reply:
I don't have a strong preference either, I was just mostly wondering why one over the other was used here.